src/components/Entry.vue

```diff
@@ -8,13 +8,20 @@
       />
     </div>
   </div>
+  <div class="flex w-full h-10 justify-evenly md:w-5/6 md:h-20 xl:w-3/4 mx-auto mt-20">
+    <template v-for="(image, index) in getGroupImages" :key="index">
+      <img :src="image" />
+    </template>
```

Comment:
So this is something I learnt recently and started using it, especially when its loops and conditional loops (using `<template>` instead of `<div>`) There is no performance difference, but `<template>` does not show up on the DOM, unlike `<div>`. This will help if you have too many conditional renders to avoid many `<div>`s in the markup. Also helps with styling, does not add unnecessary margins.